### ISSUE TITLE
sumtable() produces LaTeX error: Missing \endgroup inserted

### ISSUE THREAD

**Issue by rossellhayes** [NickCH-K/vtable#11]
Using `sumtable()` to produce LaTeX output gives an error when knitting:

Minimal example:

``````
---
title: "Untitled"
author: "Alexander Rossell Hayes"
date: "2/16/2021"
output: pdf_document
---

```{r include = FALSE}
library(knitr)
library(kableExtra)
library(vtable)
```


```{r results = "asis"}
sumtable(mtcars, out = "latex")
```
``````

Knitting this document produces the following error:
```
! Missing \endgroup inserted.
<inserted text> 
                \endgroup 
l.158 }
```

The same error occurs when manually copying and pasting the LaTeX output of `sumtable()`.

**Comment by NickCH-K** [comment]
Hmm! I will take a look at this. Thanks for the Issue.

On Feb 16, 2021 6:11 PM, Alex Rossell Hayes <notifications@github.com> wrote:

Using sumtable() to produce LaTeX output gives an error when knitting:

Minimal example:

---
title: "Untitled"
author: "Alexander Rossell Hayes"
date: "2/16/2021"
output: pdf_document
---

```{r include = FALSE}
library(knitr)
library(kableExtra)
library(vtable)
```


```{r results = "asis"}
sumtable(mtcars, out = "latex")
```


Knitting this document produces the following error:

! Missing \endgroup inserted.
<inserted text>
                \endgroup
l.158 }


The same error occurs when manually copying and pasting the LaTeX output of sumtable().

**Comment by rossellhayes** [comment]
Thank you!

In the meantime, using `out = "kable"` does knit to PDF successfully. That takes care of my use case, but still may be a problem for someone who needs real LaTeX code.

**Comment by NickCH-K** [comment]
That touches on another problem already on my list, which is that it should be defaulting to kable inside of Rmarkdown anyway but that feature has always been a bit touchy.

On Feb 16, 2021 6:25 PM, Alex Rossell Hayes <notifications@github.com> wrote:

Thank you!

In the meantime, using out = "kable" does knit to PDF successfully. That takes care of my use case, but still may be a problem for someone who needs real LaTeX code.

**Comment by NickCH-K** [comment]
Should be fixed with e93858bb38be79052a6b12ca4de670c682518d01 let me know.

Also should default consistently to `kable` in RMarkdown without any `out` specified (and improved the HTML look of kables).
